sound/hybrid_font.h

```diff
@@ -181,6 +185,17 @@ class HybridFont : public SaberBase {
 	  drag.files_found()) {
 	e = &drag;
       }
+	    
+      // ultra - dynalock lockon
+      if (lockon.files_found()) {
+        PlayPolyphonic(&lockon);
```

Comment:
This is not correct, as it plays the lockon and lock sounds simultaneously.
What we need to do is to grab a lock player, then call Play(&lockon) and PlayLoop(&lock) on it.
This will make it play lockon first and seamlessly transition to lock after that.
The code starting on line 199 below basically does this already, we just have to change the effect passed in to PlayPolyphonic()

Reply:
do i have to be a part of every change you do tho? i prefer if people just take it from there and do what they want instead of being involved since i am on a totally different path myself. i cant code 2 things i dont have the time. just to be clear, thanks!